src/models/OtrStore.js

```diff
@@ -0,0 +1,67 @@
+var _ = require("lodash");
+var DSA = require('otr').DSA;
+var OTR = require('otr').OTR;
+var Chan = require("./chan");
+
+module.exports = OtrStore;
+
+/** Stores OTR state for a given user
+ *
+ * Handels private key, current sessions and fingerprints.
+ */
+function OtrStore(name, client) {
+	_.merge(this, {
+		name: name,
+		privKey: undefined,
+		sessions: {},
+		client: client,
+	});
+	console.log('Generating DSA....');
+	this.privKey = new DSA();
+	console.log('OK, key generated');
```

Comment:
Yep, anyway, it's not something to leave here :) But for now, I want to keep an eye on how long it takes and if the time taken is sometimes significantly longer. (as far as now, the answer is no).